hazelcast/src/test/java/com/hazelcast/nio/tcp/PacketFilter.java

```diff
@@ -25,6 +25,6 @@
  */
 public interface PacketFilter {
 
-    boolean allow(Packet packet, Address endpoint);
+    boolean filter(Packet packet, Address endpoint);
```

Comment:
What does filter mean? True means the packet is OK or true means the packet should be dropped?